bip-musig2.mediawiki

```diff
@@ -139,6 +139,7 @@ In general, MuSig2 signers are stateful in the sense that they first generate ''
 However, it is possible for one of the signers to be stateless.
 This signer waits until it receives the ''pubnonce'' of all the other signers and until session parameters such as a message to sign, public keys, and tweaks are determined.
 Then, the signer can run ''NonceGen'', ''NonceAgg'' and ''Sign'' in sequence and send out its ''pubnonce'' along with its partial signature.
+Stateless signers may want to consider signing deterministically (see [[#modifications-to-nonce-generation|Modifications to Nonce Generation]]) to remove the reliance on the random number generator in the ''NonceGen'' algorithm.
```

Comment:
1. I think that signers who _do have_ access to a secure RNG may also want to sign deterministically because it protects them if the RNG turns out to be not so secure.
2. The paragraph says "However, it is possible for one of the signers to be stateless. [...] Stateless signers may want to consider signing deterministically (see [[#modifications-to-nonce-generation|Modifications to Nonce Generation]])" which seems to already indicate that this only applies to a single signer.

Reply:
Oops, I missed the non-highlighted part